### PR TITLE
Adding Che 7 Preview stacks

### PR DESCRIPTION
**Pull request by l0rd**
### What does this PR do?

Add a couple of Che 7 stacks: a first one to simple use a che7 workspace and add plugins, a second one with the tooling needed to build and publish a plugin.

### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/11502

### How have you tested this PR?

I have run them on openshift.io.

### CODE REVIEW

**Comment by l0rd** [comment]
cc @skabashnyuk

**Comment by benoitf** [comment]
@l0rd you need to apply formatter. Let me know if you can't do it

[ERROR] Failed to execute goal com.mycila:license-maven-plugin:3.0:check (check-headers) on project fabric8-ide-stacks: Some files do not have the expected license header -> [Help 1]

File: assembly/fabric8-stacks/src/main/resources/stacks.json [hunk 1]
```diff
@@ -64,6 +64,152 @@
...
+    ],
+    "workspaceConfig": {
+      "name": "che7-preview-plugin-dev",
+      "defaultEnv": "default",
+      "environments": {
+        "default": {
+          "machines": {
+            "ws/dev": {
+              "attributes": {
+                "memoryLimitBytes": "536870912"
```
Comment (garagatyi) [inline assembly/fabric8-stacks/src/main/resources/stacks.json]: 500+ MB might be too much. We would want to have extra space for several sidecars. WDYT?

File: assembly/fabric8-stacks/src/main/resources/stacks.json [hunk 2]
```diff
@@ -64,6 +64,152 @@
...
+    ],
+    "workspaceConfig": {
+      "name": "che7",
+      "defaultEnv": "default",
+      "environments": {
+        "default": {
+          "machines": {
+            "ws/dev": {
+              "attributes": {
+                "memoryLimitBytes": "536870912"
```
Comment (garagatyi) [inline assembly/fabric8-stacks/src/main/resources/stacks.json]: We don't need both an attribute and memory limit in the recipe. Can we remove one of them?

File: assembly/fabric8-stacks/src/main/resources/stacks.json (hunk 1 above)
Comment (garagatyi) [reply assembly/fabric8-stacks/src/main/resources/stacks.json]: The same for a second stack

File: assembly/fabric8-stacks/src/main/resources/stacks.json (hunk 2 above)
Comment (garagatyi) [reply assembly/fabric8-stacks/src/main/resources/stacks.json]: The same for a second stack

**Comment by l0rd** [comment]
@garagatyi I have already merged. I need to do another PR anyway to update those stacks in the next days and will tests your suggestions (reduce memory and remove the attribute).

**Pull request merged**